pkg/scaffold/init.go

```diff
@@ -77,7 +77,7 @@ func (s *initScaffolder) Scaffold() error {
 	switch {
 	case s.config.IsV1():
 		return s.scaffoldV1()
-	case s.config.IsV2():
+	case s.config.IsV2(), s.config.IsV3():
 		return s.scaffoldV2()
```

Comment:
To keep it simple I am ok with v2,v3 return scaffoldV2. 
We can change it in the future.